runner.sh

```diff
@@ -38,7 +38,7 @@ doesnt_contain_max_epochs=("${doesnt_contain_max_epochs[@]}" tcia_csv_processing
 doesnt_contain_max_epochs=("${doesnt_contain_max_epochs[@]}" transform_visualization.ipynb)
 doesnt_contain_max_epochs=("${doesnt_contain_max_epochs[@]}" 2d_inference_3d_volume.ipynb)
 doesnt_contain_max_epochs=("${doesnt_contain_max_epochs[@]}" resample_benchmark.ipynb)
-doesnt_contain_max_epochs=("${doesnt_contain_max_epochs[@]}" infoANDinference.ipynb)
```

Comment:
it becomes inaccurate after this change because in the deepgrow folder there is an `inference.ipynb`, it has `max_epochs` https://github.com/Project-MONAI/tutorials/blob/main/deepgrow/ignite/inference.ipynb